settings.gradle.kts

```
@@ -29,3 +29,4 @@ buildCache {
 }
 
 include(":app")
```

nit: `include`가 가변 인자인데 이를 활용하는 건 어떨까요?
```suggestion
include(
  ":app",
  ":waffle",
)
```

---

제안 감사합니다!